dwave/cloud/config.py

```diff
@@ -48,171 +48,141 @@
-    >>> client_qpu1 = Client.from_config(profile='primary-qpu')    # doctest: +SKIP
-    >>> client_qpu1 = Client.from_config(profile='backup-qpu')    # doctest: +SKIP
-    >>> client_sw1 = Client.from_config(profile='sw-solver')   # doctest: +SKIP
-    >>> client_qpu1.default_solver   # doctest: +SKIP
-    'EXAMPLE_2000Q_SYSTEM_A'
-    >>> client_qpu2.endpoint   # doctest: +SKIP
-    'https://url.of.some.dwavesystem.com/sapi'
-    >>> # code that uses client
-    >>> client_qpu1.close() # doctest: +SKIP
-    >>> client_qpu2.close() # doctest: +SKIP
-    >>> client_sw1.close() # doctest: +SKIP
-
-    This fourth example loads configurations auto-detected in more than one configuration
-    file, with the higher priority file (in the current working directory) supplementing
-    and overriding values from the lower priority user-local file. After instantiation,
-    an endpoint from the default section and client from the profile section is provided
-    from the user-local ``/usr/local/share/dwave/dwave.conf`` file::
+    and (2), a ``./dwave.conf`` file in the current working directory::
 
-        [defaults]
-        solver = {"qpu": true}
-
-        [dw2000]
-        endpoint = https://int.se.dwavesystems.com/sapi
-        token = ABC-123456789123456789123456789
-
-    A solver is supplemented from the file in the current working directory, which also
-    overrides the token value. ``./dwave.conf`` is the file in the current directory::
-
-        [dw2000]
+        [advantage]
         token = DEF-987654321987654321987654321
 
+    The code below supplements the API token from higher priority file (the 
+    ``./dwave.conf`` file in the current working directory), overriding the value
+    from the ``[defaults]`` and first (``[advantage]``) sections of the 
+    lower-priority user-local file, ``/usr/local/share/dwave/dwave.conf``. Use
+    of the :func:`~dwave.cloud.client.Client.get_solver` method would select 
+    an Advantage from Leap's European region using the ``DEF-987 ...`` token.
+
     >>> from dwave.cloud import Client
     >>> client = Client.from_config()  # doctest: +SKIP
-    >>> client.default_solver   # doctest: +SKIP
-    'EXAMPLE_2000Q_SYSTEM_A'
-    >>> client.endpoint  # doctest: +SKIP
-    'https://int.se.dwavesystems.com/sapi'
-    >>> client.token  # doctest: +SKIP
-    'DEF-987654321987654321987654321'
+    >>> print(client.endpoint)      # doctest: +SKIP
+    https://eu-central-1.cloud.dwavesys.com/sapi/v2/
+    >>> print(client.token)  # doctest: +SKIP
+    DEF-987654321987654321987654321
     >>> # code that uses client
     >>> client.close() # doctest: +SKIP
 
-    The next example uses :func:`~dwave.cloud.config.load_config` to load profile values.
-    **Most users do not need to use this method.** It loads from the following configuration
-    file, dwave_c.conf, located in the current working directory, and specified explicitly::
+    *   **Advanced Example:** Loading profile values
```

Comment:
I would just drop this complete section. Looks too complicated to be useful, as it's artificial and not describing a use case someone might have in practice (you would never override the envvar from code when you can just use kwargs to client init).

Reply:
Dropped